helm/validator/values.yaml

```diff
@@ -0,0 +1,366 @@
+chain:
+  name: premainnet
+  era: 21
+  chain_id: 21
+
+imageTag: release-1.5_b0cbf54f
+
+validator:
+  name:
+  image:
+    repo: diem/validator
+    tag:
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 4
```

Comment:
This comment is a strong recommendation from past experience.
Either remove the limit, or set the request to be equal to the limit.  It will save the person troubleshooting cpu related issues many hours.  Though I feel like I could write an essay about this topic, someone has already done it for me: https://amixr.io/blog/what-wed-do-to-save-from-the-well-known-k8s-incident/

Please review it and let me know what you think.

Reply:
Good point. updated the config